torchvision/models/feature_extraction.py

```diff
@@ -172,8 +175,25 @@ def _warn_graph_differences(train_tracer: NodePathTracer, eval_tracer: NodePathT
     warnings.warn(msg + suggestion_msg)
 
 
+def _get_leaf_modules_for_ops():
+    members = inspect.getmembers(torchvision.ops)
+    result = []
+    for _, obj in members:
+        if inspect.isclass(obj) and issubclass(obj, torch.nn.Module):
+            result.append(obj)
```

Comment:
Future Nit: Perhaps a registration mechanism would be useful here. We got one at the new Datasets and one planned for Models. Perhaps operators could benefit from it (TBD).